backend/src/core/middlewares/campaign.middleware.ts

```diff
@@ -22,6 +25,11 @@ const canEditCampaign = async (
     if (!hasJob && !csvStatus?.isCsvProcessing) {
       return next()
     }
+    logger.error({
```

Comment:
I think the main point of logging is for troubleshooting errors. In this case, and also for other transactions, perhaps you could consider whether the data that you've logged would be helpful for debugging, or adds extra information to the http status of the request that would have been logged with morgan.